src/js/pages/Topic.js

```diff
@@ -2,41 +2,50 @@ import React from 'react';
 import { withRouteData } from 'react-static';
 import { get } from 'lodash';
 import { cleanServiceLinks } from 'js/helpers/cleanData';
+import PropTypes from 'prop-types';
 
 // TODO: this jsonFileData is temporary. Add it to Wagtail API
 import jsonFileData from '__tmpdata/pages';
-import Hero from 'js/modules/Hero';
 import TileGroup from 'js/modules/TileGroup';
+import PageBreadcrumbs from 'js/modules/PageBreadcrumbs';
+import PageHeader from 'js/modules/PageHeader';
+import SectionHeader from 'js/modules/SectionHeader';
 import FormFeedback from 'js/page_sections/FormFeedback';
 import ThreeOneOne from 'js/page_sections/ThreeOneOne';
 
 const Topic = ({ topic }) => {
   const title = get(topic, "text", null);
```

Comment:
Super minor, but I went back to destructuring like you recommended elsewhere, when the object isn't terribly nested so we could remove lodash as a dependency where possible. 
how do you feel about that?
const { title, callToAction, ... } = topic?

Reply:
es6 ftw!!!!